Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -10353,6 +10376,42 @@ public void Actions_ImprovedShortcutSupport_ConsumesWASD(bool shortcutsEnabled)
         }
     }
 
+    [Test]
+    [Category("Actions")]
+    public void Actions_ActiveBindingIndex_PointsAtRelativeIndexInActionBindingsArray()
```

Comment:
Good to see new test but I'm not clear its related to new actions/device API code